src/hooks/useTypedSelector.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line
```

Comment:
Why do we need the eslint-disable-next-line here?

Reply:
I have the "@typescript-eslint/no-restricted-imports" rule  with a custom warning for the useSelector and useDispatch hooks.

Error: "Use typed hooks 'useTypedSelector' and 'useActions' instead."
Hint that you need to use these typed hooks. But here I have to use useSelector.